src/util/Util.js

```diff
@@ -64,6 +64,7 @@ class Util {
   static splitMessage(text, { maxLength = 2000, char = '\n', prepend = '', append = '' } = {}) {
     text = Util.resolveString(text);
     if (text.length <= maxLength) return [text];
+    if (Array.isArray(char)) char = char.filter(splitChar => text.includes(splitChar))[0];
```

Comment:
```suggestion
    if (Array.isArray(char)) char = char.find(splitChar => text.includes(splitChar));
```
No need to use `Array.prototype.filter()` just to get the first result.

Reply:
Fixed, thanks